lib/ssz_types/beacon_chain/deposit.ex

```diff
@@ -9,6 +9,16 @@ defmodule SszTypes.Deposit do
     :data
   ]
 
+  @schema [
+    %{proof: %{type: :list, schema: %{type: :bytes, size: 32}, max_size: 33, is_variable: false}},
```

Comment:
Vectors are the ones with fixed-size (unless inner values are variable-size).

```suggestion
    %{proof: %{type: :vector, schema: %{type: :bytes, size: 32}, max_size: 33}},
```